GUI/Types/Renderer/PhysSceneNode.cs

```diff
@@ -177,23 +190,34 @@ public static IEnumerable<PhysSceneNode> CreatePhysSceneNodes(Scene scene, PhysA
 
                     // meshes can be large, so ensure capacity up front
                     verts[collisionAttributeIndex].EnsureCapacity(baseVertex + vertices.Length);
-                    inds[collisionAttributeIndex].EnsureCapacity(inds[collisionAttributeIndex].Count + triangles.Length * 6);
+                    inds[collisionAttributeIndex].EnsureCapacity(inds[collisionAttributeIndex].Count + triangles.Length * 6); // TODO account for duplication from normals
```

Comment:
Can you fix this pls?

Reply:
Actually was already correct, removed the comment